unlock-app/src/utils/withConfig.js

```diff
@@ -1,24 +1,53 @@
 import React from 'react'
 import configure from '../config'
+import { WrongNetwork, MissingProvider } from '../components/creator/FatalError'
+import { ETHEREUM_NETWORKS_NAMES } from '../constants'
+import { connect } from 'react-redux'
 
 /**
  * Function which creates higher order component with the config
  * Taken from https://reactjs.org/docs/context.html#consuming-context-with-a-hoc
  */
 
-const config = configure(global)
+const isServer = typeof window === 'undefined'
+const config = !isServer ? configure(global) : {}
 const ConfigContext = React.createContext(config)
 
 // This function takes a component...
 export function withConfig(Component) {
   // ...and returns another component...
-  return function componentWithConfig(props) {
+  function componentWithConfig(props) {
     // ... and renders the wrapped component with the context config!
     // Notice that we pass through any additional props as well
+
+    const reduxStore = props.store
+    const router = props.router
+
+    if (!isServer) {
```

Comment:
Add error state for every page wrapped in `withConfig` if the config is not loaded.